assets/dev/js/editor/components/icons-manager/icons-manager.js

```diff
@@ -0,0 +1,71 @@
+import ModalLayout from './modal-layout';
+
+export default class extends elementorModules.Module {
+	onInit() {
+		this.layout = new ModalLayout();
+		this.layout.getModal().addButton( {
```

Comment:
Probably the layout itself should build this button after modal first initialization, instead of an external access.